src/coreclr/jit/emitloongarch64.cpp

```diff
@@ -5960,13 +5946,26 @@ void emitter::emitDispInsHex(instrDesc* id, BYTE* code, size_t sz)
     }
 }
 
+/*****************************************************************************
+ *
+ * For LoongArch64, the `emitter::emitDispIns` only supports
+ * the `COMPlus_JitDump` and `DOTNET_JitDump`.
```

Comment:
```suggestion
 * the `DOTNET_JitDump`.
```

We avoid introducing new mentions of `COMPlus`

Reply:
OK, thanks